libnmstate/sysctl.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+import errno
+
+
+def enable_ipv6(dev):
+    disable_ipv6(dev, disable=False)
+
+
+def disable_ipv6(dev, disable=True):
+    try:
+        with open('/proc/sys/net/ipv6/conf/%s/disable_ipv6' % dev, 'w') as f:
+            f.write('1' if disable else '0')
+    except IOError as e:
+        if e.errno == errno.ENOENT and disable:
+            # IPv6 stack is (already) not available on this device
+            return
+        raise
+
+
+def is_disabled_ipv6(dev='default'):
```

Comment:
It would be more natural to call this `is_ipv6_disabled()`. Also it should probably return a boolean instead of the number.

Reply:
Done